### PR TITLE
smartparens defaults for scala-mode

### PR DESCRIPTION
**Pull request by gregnwosu**
noddy copy of python-fix for the 
{|} this.that.theother     -> {this. that.}  theother
issue

### CODE REVIEW

File: smartparens-scala.el [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+;;
+;; For more info, see github readme at
+;; https://github.com/Fuco1/smartparens
+
+;;; Code:
+
+(require 'smartparens)
+
+;; Scala has no sexp suffices.  This fixes slurping
+;; (|sys).path.append---the dot should not travel with the closing
```
Comment (Fuco1) [inline smartparens-scala.el]: Is this valid scala example? If not, put something relevant here.

File: smartparens-scala.el [hunk 2]
```diff
@@ -0,0 +1,57 @@
+;;; smartparens-scala.el --- Additional configuration for Scala based modes.
+
+;; Copyright (C) 2015 Matus Goljer
```
Comment (Fuco1) [inline smartparens-scala.el]: Your name here :)

File: smartparens-scala.el [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+;; Author: Matus Goljer <matus.goljer@gmail.com>
```
Comment (Fuco1) [inline smartparens-scala.el]: And here.

**Comment by gregnwosu** [comment]
added changes as per comments

**Comment by Fuco1** [comment]
Thanks!

**Pull request merged**